README.md

```diff
@@ -1,70 +1,41 @@
-# Getting Started with Create React App
+# Noe Tube 
+[![Open Source Love](https://badges.frapsoft.com/os/v2/open-source.svg?v=103)](https://github.com/smilegupta)
+[![Omkar.Pro](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/omkarborude) [![Made For Neog](https://img.shields.io/badge/Made%20For-Neog-orange.svg)](https://github.com/omkarborude)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+🏆 Neo Tue is Video Library Web App.
 
-In the project directory, you can run:
```

Comment:
I read your 90% of your whole code base for more than a hour , everything looks good and clean